dev/com.ibm.ws.security.acme/src/com/ibm/ws/security/acme/internal/AcmeConfig.java

```diff
@@ -506,6 +582,17 @@ private void setValidFor(Long validForMs) {
 			this.validForMs = validForMs;
 		}
 	}
+	
+	/**
+	 * @return the renewBeforeExpirationMs
+	 */
+	public Long getRenewBeforeExpirationMs() {
+		return renewBeforeExpirationMs;
+	}
+	
+	public boolean isAutoRenew() {
```

Comment:
Javadoc.

The autoRenew being turned on or off.. does this only govern expiration? I assume so, but it isn't clear by the name of this method.

Reply:
Updated